Sources/Sentry/SentryTracer+Private.h

```diff
@@ -0,0 +1,16 @@
+#import "SentryTracer.h"
+
+@interface
+SentryTracer ()
+
+/**
+ * When an app launch is traced, after building the app start spans, the tracer's start timestamp is
+ * adjusted backwards to be the start of the first app start span. But, we still need to know the
+ * real start time of the trace for other purposes. This property provides a place to keep it before
+ * reassigning it.
+ */
+@property (strong, nonatomic) NSDate *originalStartTimestamp;
```

Comment:
l: Maybe make this a calculated property like this: 

``` Obj-c
- (NSDate*) originalStartTimestamp {
    return self.startTimeChanged ? _originalStartTimestamp  : self.startTimestamp;
}
```

so we don't have to use this if-ternary every time this property is needed, as you did in the SentryProfiler

Reply:
Yes, good call 👍🏻